koku/cost_models/serializers.py

```diff
@@ -294,10 +310,10 @@ class Meta:
     def metric_map(self):
         """Map metrics and display names."""
         metric_map_by_source = defaultdict(dict)
-        metric_map = CostModelMetricsMap.objects.all()
+        metric_map = get_json(os.path.join(BASE_DIR, "api/metrics/data/cost_models_metric_map.json"))
 
         for metric in metric_map:
-            metric_map_by_source[metric.source_type][metric.metric] = metric
+            metric_map_by_source[metric.get("source_type", None)][metric.get("metric", None)] = metric
```

Comment:
If either of those .get() calls returns a None this will probably blow up. Since you're hard referencing the key with `[]` on metric_map_by_source

Reply:
This comment can be resolved due to a slack conversation:

> Nick Bonilla  15:42
> ok, so we don't need error handling, is what I'm drawing from this, but we do need a unit test to ensure the json contains the required data






> aberglun:bulbasaur:
> 15:43
> Yeah, I would feel good about that validation.